app/src/main/java/com/example/mywatchlist/MainActivity.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.mywatchlist
```

Comment:
Please rename dirs to `.../app.mywatchlist/...` instead of `.../com/example/mywatchlist/...`. Then we don't have all these changes here and I don't have that many merge conflicts